ndb/tests/unit/test_client.py

```diff
@@ -0,0 +1,72 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import contextlib
+import pytest
+
+from unittest import mock
+
+from google.auth import credentials
+from google.cloud import environment_vars
+from google.cloud.ndb import client as client_module
+
+
+@contextlib.contextmanager
+def patch_credentials(project):
+    creds = mock.Mock(spec=credentials.Credentials)
+    patch = mock.patch("google.auth.default", return_value=(creds, project))
+    with patch:
+        yield creds
+
+
+class TestClient:
+    @staticmethod
+    def test_constructor_no_args():
+        with patch_credentials("testing"):
+            client = client_module.Client()
+        assert client.SCOPE == ("https://www.googleapis.com/auth/datastore",)
```

Comment:
Is this an instance attribute named `SCOPE`?

Reply:
It's a class attribute. Ends up getting used by the super-class when authenticating, I believe.